packages/ibm-products-styles/src/components/Saving/_saving.scss

```diff
@@ -12,6 +12,7 @@
 @use '../../global/styles/project-settings' as c4p-settings;
 
 $block-class: #{c4p-settings.$pkg-prefix}--saving;
+// $carbon-prefix: 'cds';
```

Comment:
if we're not using this we should just remove it

Reply:
Removed the comment @davidmenendez